test_poisson.py

```diff
@@ -0,0 +1,21 @@
+import numpy as np
```

Comment:
``statsmodels`` provides $\ell_1$ regulized  [Poisson regressor](https://www.statsmodels.org/dev/generated/statsmodels.discrete.discrete_model.Poisson.fit_regularized.html#statsmodels.discrete.discrete_model.Poisson.fit_regularized).
We better unittest against them.